src/Microsoft.Framework.DesignTimeHost/DesignTimeAssemblyLoadContextFactory.cs

```diff
@@ -62,9 +57,12 @@ private IAssemblyLoadContextFactory GetRuntimeFactory(IServiceProvider servicePr
 
                 // Add a cache dependency on restore complete to reevaluate dependencies
                 ctx.Monitor(_namedDependencyProvider.GetNamedDependency(_project.Name + "_Dependencies"));
-
-                return new AssemblyLoadContextFactory(applicationHostContext.ServiceProvider);
+                
```

Comment:
I told you: https://visualstudiogallery.msdn.microsoft.com/bcb3ef8e-a3a4-401e-9e63-fdf00b45268f and you will never miss a trailing whitespace...